package/plugin-gargoyle-i18n-Portuguese-BR/files/www/i18n/Portuguese-BR/openvpn.js

```diff
@@ -81,7 +81,8 @@ ovpnS.RunC="Ativo, Conectado";
 ovpnS.RunNC="Ativo, Desconectado";
 ovpnS.RunNot="Inativo";
 ovpnS.IntIP="IP Interno (Sub-rede Roteada)";
-ovpnS.CfgCredF="Credenciais\ne Arquivo de Configuração";
+ovpnS.CfgCredFM="Credenciais\ne Arquivo de Configuração\n(Múltiplo)";
+ovpnS.CfgCredFS="Credenciais\ne Arquivo de Configuração\n(Solteiro .ovpn)";
```

Comment:
Minor correction here:
From
ovpnS.CfgCredFS="Credenciais\ne Arquivo de Configuração\n(Solteiro .ovpn)";
To
ovpnS.CfgCredFS="Credenciais\ne Arquivo de Configuração\n(Único .ovpn)";